src/utils/mapRequirement.ts

```diff
@@ -29,8 +34,6 @@ const mapRequirement = (requirement?: Requirement) => {
   // Removing attributes which we don't need inside the form
   delete newRequirement.createdAt
   delete newRequirement.updatedAt
-  delete newRequirement.symbol
```

Comment:
Hmm, not sure if we can remove these lines, since we actually don't need these props on the client side, but we'll get them from the API for some requirements (e.g. NFT & Token requirements)